source/features/confirm-too-short-comments.tsx

```diff
@@ -0,0 +1,37 @@
+// import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+
+function handleIssueComment(event: delegate.Event<MouseEvent, KeyboardEvent, HTMLAnchorElement>): void {
+    let body = document.querySelector('textarea[name="comment[body]"], textarea#issue_body')?.value.length;
+	let title = document.querySelector('input#issue_title')?.value.length;
+	if ( ((body && body < 3) || (title && title < 3)) && !confirm('Less than 3 chrs, are you sure?') ) {
+			event.preventDefault();
+			event.stopImmediatePropagation(); // I don't know if it's necessary to stop comments, they're sent via ajax
+	}
+}
+
+function init(): void {
+
+	const formComment = 'form.js-new-comment-form';
+	const formNewIssue = 'form.new_issue';
+	
+	if (pageDetect.isIssue()){
+		document.querySelector(formComment)?.addEventListener('submit', handleIssueComment );
+	} else if (pageDetect.isNewIssue()){
+		document.querySelector(formNewIssue)?.addEventListener('submit', handleIssueComment );
+	}
+
```

Comment:
This should be a single event listener too, added via `delegate`, likely

```js
delegate(document, 'the 2 form selectors', submit, handleIssueComment)

Reply:
I hope I got it right: [`a801f49` (#4123)](https://github.com/sindresorhus/refined-github/pull/4123/commits/a801f4954913c7ec9f5d903f35f52c8b010e7c5e#diff-a76df45996e6ce7cc1c7d28ea7ec78e81e7fdca5cac889471cef89ab241dc55cL18-R18)